app/src/test/java/nodomain/freeyourgadget/gadgetbridge/service/TestDeviceService.java

```diff
@@ -29,7 +29,7 @@ protected void invokeService(Intent intent) {
         // calling though to the service natively does not work with robolectric,
         // we have to use the ServiceController to do that
         service.onStartCommand(intent, Service.START_FLAG_REDELIVERY, (int) (Math.random() * 10000));
-//        super.invokeService(intent);
+        super.invokeService(intent);
```

Comment:
Did you check that this works locally (without emulator or device)? See the comment above.